packages/node-cli/README.md

```diff
@@ -89,7 +89,7 @@ suitecloud account:setup
 
 ## Release Notes & Documentation
 To read the 2021.1 NetSuite's release notes and documentation, check the following sections of NetSuite's Help Center:
-- Read the latest updates under SuiteCloud SDK in the [Weekly Update section](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_158955980607.html).
+- Read the latest updates under SuiteCloud SDK in the [Help Center Weekly Updates](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_3798389663.html).
```

Comment:
Are we changing the root part of the link on purpose?
from (1): `https://system.netsuite.com/app/help/helpcenter.nl?fid=`
to (2): `https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=`
The other links have (2).

Reply:
Fixed, @davidecorreu, you were right.

Reply:
👍